frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureReleasePlanCard/FeatureReleasePlanCard.tsx

```diff
@@ -20,6 +20,23 @@ const StyledName = styled(StringTruncator)(({ theme }) => ({
     marginBottom: theme.spacing(0.5),
 }));
 
+const CardContent = styled('div')(({ theme }) => ({
+    width: '100%',
+    transition: 'opacity 0.2s ease-in-out',
+}));
+
+const HoverButtonsContainer = styled('div')(({ theme }) => ({
+    position: 'absolute',
+    top: '50%',
+    right: theme.spacing(2),
+    transform: 'translateY(-50%)',
```

Comment:
Why do you need to 50% and then translate -50%? Wouldn't `top: 0` do the same thing here?

Reply:
Good catch, was playing around with this a bit, so this was leftover.